src/main/kotlin/racingcar/IntRandom.kt

```diff
@@ -0,0 +1,11 @@
+package racingcar
+
+import java.util.Random
```

Comment:
코틀린의 `Random`을 사용하면 어떨까요?

```suggestion
import kotlin.random.Random
```

Reply:
코틀린 랜덤 클래스 사용법을 알게되었습니다.
48c839a6042d43bfa6295713a75e8ae1095da5cc 에서 적용했습니다.